### ISSUE TITLE
Demo fails silently on iPad

### ISSUE THREAD

**Issue by pannous** [Ameobea/web-synth#21]
Just a black screen

**Comment by pannous** [comment]
https://synth.ameo.dev/fm.html loads but plays no sound

**Comment by Ameobea** [comment]
Hi, thanks for the report.  Were you by chance using the Safari browser?  I checked and found that I was emitting some WebAssembly SIMD opcodes that Safari couldn't handle; I've since fixed that.  The FM synth demo now works when I test it in Safari on Mac OS.

However, loading different compositions such as https://synth.ameo.dev/composition/50 seems to be causing a huge memory leak that only happens in Safari.  On Chrome, the webpage uses ~330MB of RAM, but on Safari I saw it grow to over 12GB.  I will have to investigate this further; I'm not sure if it's an issue with Safari, a bug in my code that only manifests on Safari, or perhaps it's the [M1 memory leaks](https://www.macworld.com/article/549755/m1-macbook-app-memory-leaks-macos.html) that people were talking about a month or so ago.

Anyway, thanks again for letting me know about that; I'll keep an eye out and look further into this.
